src/js/views/forms/form/form.scss

```diff
@@ -137,6 +137,7 @@
   .icon {
     color: $black-60;
     font-size: 20px;
+    vertical-align: middle;
```

Comment:
Better aligns the action buttons (e.g expand, show sidebar, etc.) with the other elements in that section.

![Screenshot 2023-03-20 at 3 53 41 PM](https://user-images.githubusercontent.com/35355575/226465072-7bb1954b-8370-4a53-9fb9-3d7483b3e576.png)